packages/types/package.json

```
@@ -1,18 +1,20 @@
 {
   "name": "@cloudinary-util/types",
   "version": "1.0.5",
-  "types": "./src/index.d.ts",
+  "type": "module",
+  "main": "./dist/index.cjs",
+  "types": "./dist/index.d.cts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    }
+  },
```

we should cover bases if building seperate esm and cjs builds but is probably not necessary to change this package at all since it's only types

```suggestion
  "main": "./dist/index.cjs",
  "module": "./dist/index.js",
  "types": "./dist/index.d.ts",
  "exports": {
    ".": {
      "require": {
        "types": "./dist/index.d.cts",
        "default": "./dist/index.cjs"
      },
      "import": {
        "types": "./dist/index.d.ts",
        "default": "./dist/index.js"
      }
    }
  },
```

---

I agree this could be simplified but I went with reusing the same build process we have across packages.

I hadn't seen this `.d.cts` setup either but it seems to be what the latest version of tsup is doing by default? If you'd like to follow up with a future PR adjusting this I think it could be improved but there are already quite a lot of changes in this PR and unfortunately I don't have additional hours to spend updating it at this point.

For now, I'm not sure if the `.d.cts` files would be resolved for `.cjs` imports or if it would just resolve to the `.d.ts` files but in any case the types should be the same in the vast majority of circumstances so I don't think it should break anything.